reports/hemang.md

```diff
@@ -192,7 +192,7 @@ The coala Online module now has support for all the bears, bears with optional a
 
 <br>
 
-coala-quickstart has got a new running mode, allow-incomplete-sections which helps users to create a coafile without omitting bears with non optional settings. 
+coala-quickstart has got a new running mode, allow-incomplete-sections which helps users to create a coafile without omitting bears with non optional settings.
```

Comment:
Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.